client/app/components/Header/__tests__/Header.spec.jsx

```diff
@@ -1,34 +1,36 @@
 // @flow
-import { mount } from 'enzyme';
+import { render, fireEvent } from '@testing-library/react';
 import React from 'react';
 import Header from '../index';
 
-const getComponent = () => (
+const getComponent = () => render(
   <Header
     home={{ name: 'Home', url: '/some-path' }}
     links={[
       { name: 'Link 1', url: '/some-path-one', active: true },
       { name: 'Link 2', url: '/some-path-two', dataMethod: 'delete' },
     ]}
-  />
+  />,
 );
 
-const wrapper = mount(getComponent());
-
 describe('Header', () => {
   it('renders correctly', () => {
-    expect(wrapper.length).toEqual(1);
+    const { container } = getComponent();
+    expect(container.firstChild).not.toBeNull();
   });
 
   it('toggles hamburger correctly', () => {
-    wrapper.find('#headerHamburger').simulate('click');
-    expect(wrapper.find('#headerMobile').length).toEqual(1);
-    wrapper.find('#headerHamburger').simulate('click');
-    expect(wrapper.find('#headerMobile').length).toEqual(0);
+    const { container, getByLabelText } = getComponent();
+    fireEvent.click(getByLabelText('expand menu'));
+    const headerMobile = container.querySelector('#headerMobile');
+    expect(headerMobile).toBeInTheDocument();
```

Comment:
Instead of testing for the presence of classes, we should be testing for text. Something like `expect(screen.getByText('Link 1')).toBeInTheDocument()` and `expect(screen.getByText('Link 2')).toBeInTheDocument()` should suffice.